### ISSUE TITLE
Adding New Account from Settings

### ISSUE THREAD

**Issue by gekhung** [zom/Zom-Android-XMPP#62]
The latest Zom Beta version on hockey app has 'Add New Account' that takes me to 'Welcome to Zom' page. Are we going to allow users to create account while they already in an account?

**Comment by n8fr8** [comment]
I am not sure... at the least, we will have "find new server" or "switch servers" if there is an issue with the users current server being blocked.

Mostly, that is in the UI to allow us to check the onboard UI without having to reset the app everytime.

**Comment by gekhung** [comment]
Right, at the moment, it feels like you can have  more than 1 accounts open. I haven't tried adding a new account. Let me try and see what happens.

**Comment by gekhung** [comment]
At the moment, after creating the new account, it's taking me to the existing account, the new account doesn't appear in 'Me' page.

**Comment by n8fr8** [comment]
I think the account is there in the background,  but there is no way to say to specifically use it.

**Comment by n8fr8** [comment]
You can now long press to set the default account, but we need to work on this more.
